godotopenxrmeta/src/main/java/org/godotengine/openxr/vendors/meta/GodotOpenXRMeta.kt

```diff
@@ -153,6 +154,11 @@ class GodotOpenXRMeta(godot: Godot?) : GodotPlugin(godot) {
         if (PermissionsUtil.hasManifestPermission(activity, FACE_TRACKING_PERMISSION)) {
             permissionsToRequest.add(FACE_TRACKING_PERMISSION)
         }
+        // Request the body tracking permission if it's included in the manifest
+        if (PermissionsUtil.hasManifestPermission(activity, BODY_TRACKING_PERMISSION)) {
+            Log.d(TAG, "Requesting permission '${BODY_TRACKING_PERMISSION}'")
+            PermissionsUtil.requestPermission(BODY_TRACKING_PERMISSION, activity)
+        }
```

Comment:
@Malcolmnixon We've changed the logic used for requesting permissions; you need to include the permission in the `permissionsToRequest` list as done above instead.

See https://github.com/GodotVR/godot_openxr_vendors/pull/102 for reference.

Reply:
Switched over to use permissionsToRequest.